deepchem/trans/transformers.py

```diff
@@ -1182,3 +1181,46 @@ def __init__(self,
   def transform_array(self, X, y, w):
     X = self.featurizer.featurize(X)
     return X, y, w
+
+
+class DataTransforms():
```

Comment:
Can you add unit tests for this class? Also, this doesn't look like a generic `DataTransforms` class; rather `ImageTransforms` or the like.